rest_framework/serializers.py

```diff
@@ -1627,6 +1633,29 @@ def get_unique_for_date_validators(self):
 
         return validators
 
+    def get_partial_update_extra_fields(self):
+        partial_update_extra_fields = getattr(self.Meta, 'partial_update_extra_fields', None)
```

Comment:
If it retrieves an empty list by default it really simplifies the rest of the code.

```suggestion
        partial_update_extra_fields = getattr(self.Meta, 'partial_update_extra_fields', [])
```